packages/client/src/generation/utils/buildInlineSchema.ts

```diff
@@ -13,9 +14,11 @@ const readFile = fs.promises.readFile
 export async function buildInlineSchema(clientEngineType: ClientEngineType, schemaPath: string) {
   if (clientEngineType === ClientEngineType.DataProxy) {
     const b64Schema = (await readFile(schemaPath)).toString('base64')
+    const schemaHash = crypto.createHash('sha256').update(b64Schema).digest('hex')
```

Comment:
To save on compute, we now do the schema hash calculation on generate instead of every time.